editor/debugger/editor_debugger_tree.cpp

```diff
@@ -147,17 +147,16 @@ void EditorDebuggerTree::update_scene_tree(const SceneDebuggerTree *p_tree, int
 
 	// Nodes are in a flatten list, depth first. Use a stack of parents, avoid recursion.
 	List<Pair<TreeItem *, int>> parents;
-	for (int i = 0; i < p_tree->nodes.size(); i++) {
+	for (const SceneDebuggerTree::RemoteNode &node : p_tree->nodes) {
 		TreeItem *parent = nullptr;
 		if (parents.size()) { // Find last parent.
-			Pair<TreeItem *, int> &p = parents[0];
+			Pair<TreeItem *, int> &p = parents.front()->get();
```

Comment:
This is used so often, could it be worth a `get_front()` function? What do you think? :thinking:

Reply:
Can see, was considering it and can look at it in a followup (this one is already a bit busy without replacing all those cases)

Reply:
Yes, I would concur a followup would be better, as this is pretty big PR to review already. Nearly half way through reading! :grin: